var/spack/repos/builtin/packages/py-torch-cluster/package.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyTorchCluster(PythonPackage):
+    """This package consists of a small extension library of
+    highly optimized graph cluster algorithms for the use in
+    PyTorch."""
+
+    homepage = "https://github.com/rusty1s/pytorch_cluster"
+    url      = "https://github.com/rusty1s/pytorch_cluster/archive/1.5.7.tar.gz"
+
+    version('1.5.7', sha256='71701d2f7f3e458ebe5904c982951349fdb60e6f1654e19c7e102a226e2de72e')
+
+    variant('cuda', default=False)
+
+    depends_on('python@3.6:', type=('build', 'run'))
+    depends_on('py-scipy', type=('build', 'run'))
```

Comment:
According to the `setup.py`, this is only needed for testing, although the `setup.py` also doesn't list pytorch as a dependency lol

Reply:
Yeah, these 5 packages that I've recently pushed up all have incomplete dependency listings in `setup.py`